examples/values-secret.yaml

```diff
@@ -17,5 +17,21 @@ secrets:
   aws:
     s3Secret: test-secret
 
-
-
+  # The cluster_xxxx pattern is used for creating externalSecrets that
+  # will be used by ArgoCD to detect other clusters.
+  #
+  # Create a service account with enough permissions and extract the token
+  #
+  # CLUSTER_TOKEN=$(oc describe secret -n default argocd-external-token | grep 'token:' | awk '{print$2}')
+  # CLUSTER_CA=$(oc extract -n openshift-config cm/kube-root-ca.crt --to=- --keys=ca.crt | base64 | awk '{print}' ORS='')
+  cluster_example:
+    name: example
+    server: https://api.example.openshiftapps.com:6443
```

Comment:
I think it would be enough with a script that creates/extracts the token and eventually adds the secret configuration from a template. It's not about the complexity of creating the secret but the playbook would require credentials to connect to the remote cluster.